static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -128,8 +133,17 @@ export class DeprecatedLine extends Component<Props, State> {
   // https://facebook.github.io/react/tips/props-in-getInitialState-as-anti-pattern.html
   state: State = {
     isExpanded: this.props.isExpanded,
+    isHovering: false,
   };
 
+  handleMouseEnter = debounce(() => {
```

Comment:
I don't think just using the `debounce` function quite works out the way we want it to. We'd like for this to work like the following:

- User hovers over frame 
- Loading state is shown immediately, but the request isn't made yet
- After 100ms, we actually start making the request

I don't think this shows the loading state immediately, and it seems like the request will still be made even if the user moves their cursor before the 100ms.